### PR TITLE
Add expiry time to token.  

### LINKED ISSUE #1415: Add expiry date time to token yaml
**Is your feature request related to a problem? Please describe.**
I sometimes create default tokens and then get distracted before using them.
When I do use them, they are invalid

**Describe the solution you'd like**
If the token yaml had more information, eg time created and token duration that would help me avoid wasting time 
think about:

- did the link fail because I've run out of time or because it was already used?
- is connectivity possible between these two sites

**Describe alternatives you've considered**
An expiry date time
No of uses 

**Additional context**
While this doesn't solve the problem of creating a link to the wrong site, because you've mixed up tokens it might be a help when the urls are long and all look the same at a glance.

### PR DESCRIPTION
**Pull request by lynnemorrison**
Add expiry time to token.  When token is being used to create a link verify the token hasn't expired.  If it has log a warning but continue with trying to bring up link.

fixes #1415

### CODE REVIEW

File: client/connector_create.go [hunk 1]
```diff
@@ -113,6 +114,17 @@ func (cli *VanClient) ConnectorCreateSecretFromData(ctx context.Context, options
 				return nil, fmt.Errorf("Claims not supported. %s", err)
 			}
 		}
+		if expirationStr, ok := secret.ObjectMeta.Annotations[types.ClaimExpiration]; ok {
+			expiration, timeErr := time.Parse(time.RFC3339, expirationStr)
+			if timeErr != nil {
+				err = timeErr
+				return nil, fmt.Errorf("timer error %s", err)
```
Comment (grs) [inline client/connector_create.go:121]: I'd suggest we just print a warning here also, e.g. 'cannot parse expiration'

Comment (lynnemorrison) [reply client/connector_create.go]: fixed

Comment (grs) [reply client/connector_create.go]: Its still returning an error rather than printing a warning.

Comment (lynnemorrison) [reply client/connector_create.go]: sorry,  just focused on what was being printed, vs type of error.   I have now fixed to just log a warning but to continue on processing.

**Review by grs — APPROVED** [review]

**Review by nluaces — APPROVED** [review]

**Comment by grs** [comment]
Thank you @lynnemorrison !

**Pull request merged**